base_app/base.py

```diff
@@ -16,10 +16,13 @@ def index():
     headers = {'authorization': 'Bearer ' + YELP_API_KEY}
     lat = request.args['latitude']
     long = request.args['longitude']
-    cuisine = request.args['categories']
-    params = {'latitude': f'{lat}', 'longitude': f'{long}', 'term': f'{cuisine}'}
     url = 'https://api.yelp.com/v3/businesses/search'
-
+    if 'price' in request.args.keys():
+        price = request.args['price']
```

Comment:
Thank you for providing the corresponding documentation that addresses the way to access dictionary keys.